crates/provider-sdk/src/provider.rs

```diff
@@ -771,6 +771,8 @@ pub struct ProviderConnection {
     link_name: String,
     provider_key: String,
 
+    // TODO: Reference this field to get static config
+    #[allow(unused)]
```

Comment:
I left this in here as I'm not 100% sure if we need it, but I was pretty sure this is the proper place for static provider config to go.